pkg/core/mpt/trie_store.go

```diff
@@ -37,10 +33,11 @@ func NewTrieStore(root util.Uint256, mode TrieMode, backed storage.Store) *TrieS
 	}
 }
 
-// Get implements the Store interface.
+// Get implements the Store interface. It can return [errors.ErrUnsupported]
+// for unsupported operations.
 func (m *TrieStore) Get(key []byte) ([]byte, error) {
 	if len(key) == 0 {
-		return nil, fmt.Errorf("%w: Get is supported only for contract storage items", ErrForbiddenTrieStoreOperation)
+		return nil, fmt.Errorf("%w: Get is supported only for contract storage items", errors.ErrUnsupported)
```

Comment:
It's a good change but it seems to me that we have a lot of other cases of "unsupported" usage. Should we consider replacing most of them?

Reply:
Maybe. It's more of a question of how to find them.